accounts.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: master
+---
+
+## Setting up accounts
+
+A couple of accounts are needed during the course. The motivation
+for this is to provide a real life hands-on experience using tools with massive
+traction and network effect.
+
+We recommend that all participants who do not yet have user accounts on the following
+platforms create new accounts before the workshop starts:
+
+- [GitHub](https://github.com/)
+- [Read the Docs](https://readthedocs.org/)
```

Comment:
people can sign in into read the docs and travis with github, so they only need a github account

Reply:
Indeed, this will simplify things. I will send a patch for this.

Reply:
Done.